### PR TITLE
Don't show transparency message on attach_receipt page when not signe…

### PR DESCRIPTION
**Pull request by exu3**
…d in

closes #2592

### CODE REVIEW

**Review by cjdenio — CHANGES_REQUESTED** [review]
Hmm, this also makes the header disappear (which might be desired)

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -57,7 +57,7 @@
         </article>
       </main>
     <% else %>
-      <%= render 'application/header' unless @no_app_shell %>
+      <%= render 'application/header' unless (@no_app_shell || @no_transparency_header) %>
```
Comment (cjdenio) [inline app/views/layouts/application.html.erb]: Oop I think you meant to put this on line 32

Comment (exu3) [reply app/views/layouts/application.html.erb]: https://github.com/hackclub/bank/pull/2633/commits/62b55ca6850908de3cc3a2ed6604890372485285 Just fixed, thanks! I should test things before I commit :festpee:

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <% title @event.name %>
 <% include_modals %>
+<% no_transparency_header %>
```
Comment (cjdenio) [inline app/views/events/show.html.erb]: hmmmmmmmm i think you meant to remove this

Comment (exu3) [reply app/views/events/show.html.erb]: Ah I did, thank you.

**Review by cjdenio — APPROVED** [review]
Looks great! Thanks for fixing this!

**Pull request merged**